f2/src/utils/formatAnalystEmail.js

```diff
@@ -40,7 +40,8 @@ const formatReportInfo = (data) => {
   delete data.reportId
   delete data.submissionTime
```

Comment:
Do we want to format the date here as well? to match other dates in the report

Reply:
oh, good catch! 👍